vincevd1/thaliany
# Change login and logout flow so people don't have to authorize again after they logout

## Summary
When logging out we used to clear the whole local storage but that meant the user had to relog again meaning a duplicate authorized app on the concrexit dashboard. Instead of this we are now not removing the refresh token and when the user wishes to login again we will just login with the refresh token

## How to test
1. Logout of thaliany
2. Clear local storage
3. Revoke all prior thaliany applications in the dashboard
4. Login to thaliany
5. Logout again
6. Login again and you will see you did not have to reauthorize the application
